components/greaselion/browser/greaselion_service_impl.cc

```diff
@@ -227,7 +227,6 @@ GreaselionServiceImpl::GreaselionServiceImpl(
     : download_service_(download_service),
       install_directory_(install_directory),
       extension_system_(extension_system),
```

Comment:
Do we still need to pass `extension_system` in ctor?
maybe replace `SetExtensionService()` to `SetExtensionSystem()`

Reply:
decided to keep changes in this PR to minimum. other improvements are possible here, but can be done as a follow-up/refactoring.